.travis.yml

```diff
@@ -48,7 +48,7 @@ install:
   - docker info
   - cd ${TRAVIS_BUILD_DIR}/pupil-spa && travis_retry yarn install --frozen-lockfile
   - cd ${TRAVIS_BUILD_DIR}/admin && travis_retry yarn install --frozen-lockfile
-  - cd ${TRAVIS_BUILD_DIR}/pupil-auth && travis_retry yarn install --frozen-lockfile
+# - cd ${TRAVIS_BUILD_DIR}/pupil-api && travis_retry yarn install --frozen-lockfile
```

Comment:
Shouldn't this be enabled and tests run on Travis?

Reply:
there are currently no tests due to the fact that there is no logic implemented yet